content/running_bookwyrm/reverse-proxy.md

```diff
@@ -112,3 +112,34 @@ If everything worked correctly, your BookWyrm instance should now be externally
 *Note: the `proxy_set_header Host $host;` is essential; if you do not include it, incoming messages from federated servers will be rejected.*
 
 *Note: the location of the ssl certificates may vary depending on the OS of your server*
+
+
+#### Caddy
+
+[Caddy](https://caddyserver.com) is an alternative reverse-proxy server that is simple to use and can manage TLS certificate renewals and static file serving automatically. This guide assumes that you already have a Caddy server running and it has access to the bookwyrm docker network.
```

Comment:
Can we clarify what "has access to the bookwyrm docker network" means in this context?